src/data/meetings.json

```diff
@@ -120,13 +120,31 @@
   "localMedia": {
     "ID": "localMedia",
     "title": "Ad Hoc Meeting Space",
-    "localVideo": {},
+    "localVideo": {
+      "muted": false
```

Comment:
Is there a reason for adding the `muted` property?

Reply:
No strong reason for that. Will rollback and focus on the sharing part.